wear/build.gradle

```diff
@@ -1,5 +1,39 @@
 apply plugin: 'com.android.application'
 
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.1'
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+project.ext{
```

Comment:
These values i set manually, but they are not correct. Should i find them somehow somewhere?